src/app/coursegrab/models/session.py

```diff
@@ -46,4 +48,5 @@ def serialize_session(self):
             "session_token": self.session_token,
             "session_expiration": round(self.session_expiration.timestamp()),
             "update_token": self.update_token,
+            "last_used": round(self.last_used.timestamp())
```

Comment:
Is there a reason you added this to `serialize_session`, not `serialize`?
`serialize_session` is used when returning the response for `/session/initialize` API  - changing this function would alter the response of this API so I was just wondering!

Reply:
hmm i put it there because i thought last used would be associated with initializing/refreshing sessions but, thinking about it now, it might make more sense to put it in the general serialize function